### ISSUE TITLE
[MAINTENANCE] Document key bindings

### ISSUE THREAD

**Issue by azriel91** [amethyst/amethyst#1331]
## Description

Currently there is no book documentation for how to specify input bindings.
There are examples with example for `axes`, but none for `actions`.

May be worth explaining how to implement your own `AX` and `AC` types instead of simply using `String`s:

```rust
(
    axes: {
        // Players are numbered using zero based indices, so player "one" is 0
        (player: 0, axis: X): Emulated(pos: Key(Right), neg: Key(Left)),
        (player: 0, axis: Z): Emulated(pos: Key(Down), neg: Key(Up)),
    },
    actions: {
        (player: 0, action: Defend): [[Key(O)]],
        (player: 0, action: Jump): [[Key(P)]],
        (player: 0, action: Attack): [[Key(LBracket)]],
        (player: 0, action: Special): [[Key(RBracket)]],
    },
)
```

## Reason

So that it is easier for users to use `Bindings` instead of traversing code.

## Impact

> Do you think this change will have a negative impact on anything else?

User sanity if not done.

## Additional Information

>  It may be worth adding a short section about ScanCode in the documentation about key bindings.
>
> &ndash; `@akabash0r#8360` (discord)

https://github.com/amethyst/amethyst/blob/47bcf8a2e5e86a4af6b8abfc189e8f0690cbd581/amethyst_input/src/button.rs#L7-L26

**Comment by jordgubben** [comment]
What has to be done, I understand this correctly, it to go through the `amethyst_input` and related examples tu understand it. Then write a short add a short chapter about input to the book while still remembering what it was like to not understand any of this.

Any tips on where to start?

**Comment by Max-Rensen** [comment]
So maybe a new chapter in the book needs to be added, which contains two sub-chapters, one on how to specify and load input bindings from a RON file and one on how to implement custom AX/AC types.

I think I am able to do this, I was looking for a good first issue in this project anyways. I don't know if @jordgubben or anyone else is working on it currently, so please notify me if I can or cannot work on this.

**Comment by azriel91** [comment]
I don't think anyone's got anything in progress, so go ahead 😃

**Comment by jordgubben** [comment]
@Max-Rensen I havn't started yet, so go ahead.

**Comment by Max-Rensen** [comment]
So I made a pull request on the matter #1728. Let me know if it's up to standards, if so I could add another subsection on window/UI events.

**Comment by azriel91** [comment]
Closed by #1728 :tada:.
